File: dynamic_listings/C++/default/containers/bounce_tween.cpp
#include<cmath>

float bounce_tween(float t){
    // This constant will allow us to overshoot the max value by around 10%
    const float c = 1.70158;

    return 1 + (c + 1) * pow(t - 1, 3) + c * pow(t - 1, 2);
}

File: dynamic_listings/python/default/containers/bounce_tween.py
def bounce_tween(t: float) -> float:
    # This constant will allow us to overshoot the max value by around 10%
    C = 1.70158

    return 1 + (C + 1) * (t - 1) ** 3 + C * (t - 1) ** 2
